### PR TITLE
19966 create cid minting test env

### PR DESCRIPTION
**Pull request by jsjiang**
@cscollett @RvanB Hi Charlie and Raiden,
There are two  types of changes:
1. Modified unit test for the local cid minter to combine create table and insert datasets into one SQL file:
* prepare_zephir_records/cid_minter/tests/test_local_cid_minter.py
* prepare_zephir_records/cid_minter/tests/test_local_cid_minter/create_cid_minting_store_table.sql
* prepare_zephir_records/cid_minter/tests/test_local_cid_minter/prepare_cid_minter_datasets.sql

2. Scripts to create databases and initiate datasets for the integration tests:
For the Local CID  minter (SQLite):
* prepare_zephir_records/integration_tests/create_local_cid_minting_store.sql
* prepare_zephir_records/integration_tests/prepare_local_minter.sh

For the Zephir database (MySQL, host=ds-d2d-htmm-dev, database name=devtst):
* prepare_zephir_records/integration_tests/create_oclc_ref_table.sql - a copy of the OCLC cluster info in levelDB. It will not be used for testing but can be used for manually exam OCLC clusters. We do not have an easy to use interface for examing datasets in leveDB.
* prepare_zephir_records/integration_tests/create_zephir_tables.sql
* prepare_zephir_records/integration_tests/prepare_zephir_db.sh

For LevelDB:
* prepare_zephir_records/integration_tests/prepare_leveldb.py
* prepare_zephir_records/integration_tests/test_datasets/leveldb_test_datasets.csv

And finally this script which combines the above create database scripts into one:
* prepare_zephir_records/integration_tests/run_integration_tests.sh

Please take a look and let me know if you have questions.

Thank you

Jing

### CODE REVIEW

File: prepare_zephir_records/integration_tests/how_to_run_integration_test.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# How to run the integration tests
+
+```
+./run_integration_tests.sh <password_for_htmmdba>"
```
Comment (RvanB) [inline prepare_zephir_records/integration_tests/how_to_run_integration_test.md:4]: In the environment where this is run, is it fine for the htmmdba password to be saved in the command line history? Would providing it via stdin be preferable?

**Review by RvanB — APPROVED** [review]
Looks good! I left one comment.

File: prepare_zephir_records/integration_tests/how_to_run_integration_test.md (hunk 1 above)
Comment (cscollett) [reply prepare_zephir_records/integration_tests/how_to_run_integration_test.md:4]: We should maybe use a locked down config file in the format mysql expects. See second, preferred answer: https://unix.stackexchange.com/questions/205180/how-to-pass-password-to-mysql-command-line

**Review by cscollett — APPROVED** [review]

File: prepare_zephir_records/integration_tests/how_to_run_integration_test.md (hunk 1 above)
Comment (jsjiang) [reply prepare_zephir_records/integration_tests/how_to_run_integration_test.md:4]: Thank you both for the suggestions. I will try the config file approach after merging these changes.
Jing

**Pull request merged**